src/test/first.nr

```diff
@@ -44,7 +44,7 @@ unconstrained fn test_end_vote() {
 #[test(should_fail)]
 unconstrained fn test_fail_end_vote_by_non_admin() {
     let (env, voting_contract_address, _) = utils::setup();
-    let alice = env.create_account(1);
+    let alice = env.create_account(2);
```

Comment:
Why are these getting bumped?

Reply:
the change was made on master to improve the TXE i guess. `yarn update` just updates to what is on master and I make sure it works with `yarn test` :stuck_out_tongue: 

It was updated in this commit: https://github.com/AztecProtocol/aztec-packages/commit/f8448bfb9d2353116e270c22449d1d1960adf683

Reply:
I see. The utilis one I had made use `0` in my PR so that the convention of `1` and `2` for further accounts could be maintained. By having utils use `1`, the other tests have to be aware that they must start from 2.
Understandable to keep here what is there, even though it's a code smell imho.

Reply:
To be clear I am just taking this from aztec-packages. Not modifying anything to keep it simple